edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java

```diff
@@ -243,17 +255,25 @@ private Level getRejuvenatedLogLevel(LinkedList<Float> boundary, LogInvocation l
 		float max = getMaxDOI(degreeOfInterests);
 		LinkedList<Float> boundary = new LinkedList<>();
 		if (min < max) {
-			if (this.useLogCategory) {
-				float interval = (max - min) / 4;
-				IntStream.range(0, 5).forEach(i -> boundary.add(min + i * interval));
-			} else if (this.useLogCategoryWithConfig) {
+			if (this.useLogCategory || this.useLogCategoryWithConfig) {
```

Comment:
This lines' logic is different from my code.

Reply:
Yes, that's the point. This isn't a refactoring.

Reply:
How do you think? Keep the previous logic or use your logic?